gui/default/syncthing/core/syncthingController.js

```diff
@@ -1176,8 +1179,8 @@ angular.module('syncthing.core')
                 return 'notify';
             }
 
-            // all used devices are paused except (this) one
-            if (pauseCount === deviceCount - 1) {
+            // all used devices are paused
+            if (pauseCount === deviceCount) {
```

Comment:
I'd argue that we might change this one to `deviceCount > 0 && pauseCount === deviceCount`.